usr/local/share/deskpro/container-var-reference.json

```diff
@@ -1,419 +1,711 @@
+    "type": "string",
     "example": "X-Forwarded-Port"
   },
   {
     "name": "HTTP_USER_REAL_PROTO_HEADER",
-    "type": "string",
     "description": "The header to use for the users real protocol.",
+    "type": "string",
     "example": "X-Forwarded-Proto"
   },
   {
     "name": "HTTP_USER_SET_HTTPS_PORT",
-    "type": "string",
     "description": "For web requests coming through a secure https connection, set the end-user port being used.",
-    "example": "12345"
+    "type": "string",
+    "example": "12345",
+    "default": "443"
   },
   {
     "name": "HTTP_USER_SET_HTTPS_PROTO",
-    "type": "string",
     "description": "For web requests coming through a secure https connection, set the end-user protocol being used.",
-    "example": "https"
+    "type": "string",
+    "example": "https",
+    "default": "https"
   },
   {
     "name": "HTTP_USER_SET_HTTP_PORT",
-    "type": "string",
     "description": "For web requests coming through a http connection, set the end-user port being used.",
-    "example": "12345"
+    "type": "string",
+    "example": "12345",
+    "default": "80"
   },
   {
     "name": "HTTP_USER_SET_HTTP_PROTO",
-    "type": "string",
     "description": "For web requests coming through a http connection, set the end-user protocol being used.",
-    "example": "http"
+    "type": "string",
+    "example": "http",
+    "default": "http"
   },
   {
     "name": "HTTP_USE_TESTING_CERTIFICATE",
+    "description": "Set true to use the built-in (non-secure) testing certificate for HTTPS/443.",
     "type": "boolean",
-    "description": "Set true to use the built-in (non-secure) testing certificate for HTTPS/443."
+    "default": "false"
   },
   {
     "name": "INSTALL_ADMIN_EMAIL",
+    "description": "Used with AUTO_RUN_INSTALLER. The email address of the initial admin user. If not provided, will default to admin@deskprodemo.com.",
     "type": "string",
-    "description": "Used with AUTO_RUN_INSTALLER. The email address of the initial admin user. If not provided, will default to admin@deskprodemo.com."
+    "default": "admin@deskprodemo.com"
   },
   {
     "name": "INSTALL_ADMIN_PASSWORD",
-    "type": "string",
-    "description": "Used with AUTO_RUN_INSTALLER. The account password of the initial admin user. If not provided, will default to a random string."
+    "description": "Used with AUTO_RUN_INSTALLER. The account password of the initial admin user. If not provided, will default to a random string.",
+    "type": "string"
   },
   {
     "name": "INSTALL_URL",
+    "description": "Used with AUTO_RUN_INSTALLER. The URL of the Deskpro instance. If not provided, will default to http://127.0.0.1/",
     "type": "string",
-    "description": "Used with AUTO_RUN_INSTALLER. The URL of the Deskpro instance. If not provided, will default to http://127.0.0.1/"
+    "default": "http://127.0.0.1"
   },
   {
-    "description": "",
-    "name": "NGINX_ERROR_LOG_LEVEL"
+    "name": "IS_EXEC_MODE",
```

Comment:
IS_EXEC_MODE is an 'internal' state var, you wouldn't set this yourself so it shouldn't be in the var reference